src/content/docs/infrastructure/amazon-integrations/manage-aws-data/aws-integrations-metrics.mdx

```diff
@@ -4574,6 +4595,162 @@ We no longer add these mappings, and we're committed to preserving all the new m
+
+<Callout variant="important">
+  This optional capability is complementary to the CloudWatch Metric Streams integration. Your service discovery and monitoring will be incomplete if you don't enable this optional capability.
+</Callout>
+
+The solution relies on [AWS Config](https://docs.aws.amazon.com/config/index.html), which might incur in additional costs in your AWS account. AWS Config provides [granular controls](https://docs.aws.amazon.com/config/latest/developerguide/select-resources.html) to determine which services and resources are recorded. New Relic will only ingest metadata from the available resources in your AWS account.
+
+The following services and namespaces are supported:
+
+* ALB/NLB
+* API Gateway (excluding API v1)
+* DynamoDB
+* EBS
+* EC2
+* ECS
+* ELB
+* Lambda
+* RDS
+* S3
+
+### Infrastructure agent metrics and EC2 metadata decoration [#infra-data-decoration]
+
+As with the EC2 API polling integration, when the infrastructure agent is installed on a host and the EC2 namespace is active via AWS CloudWatch metric stream integration, then all the infrastructure agent events and metrics are decorated with additional metadata.
+
+The following attributes will decorate infrastructure samples. Some of these may not be applicable on all environments: `awsAvailabilityZone`, `ec2InstanceId`, `ec2PublicDnsName`, `ec2State`, `ec2EbsOptimized`, `ec2PublicIpAddress`, `ec2PrivateIpAddress`, `ec2VpcId`, `ec2AmiId`, `ec2PrivateDnsName`, `ec2KeyName`, `ec2SubnetId`, `ec2InstanceType`, `ec2Hypervisor`, `ec2Architecture`, `ec2RootDeviceType`, `ec2RootDeviceName`, `ec2VirtualizationType`, `ec2PlacementGroupName`, `ec2PlacementGroupTenancy`.
+
+## Custom metrics and percentiles [#custom-metrics-percentiles]
+
+The CloudWatch metric stream integration automatically ingests new metrics configured in the stream, including custom metrics and percentiles.
+
+### Custom metrics [#custom-metrics]
+
+To ingest CloudWatch custom metrics, make sure your custom namespace is visible in the CloudWatch metric stream configuration and it's not being filtered by inclusion or exclusion rules.
+
+### Percentiles [#percentiles]
+
+AWS CloudWatch allows you to [define additional statistics](https://aws.amazon.com/about-aws/whats-new/2022/04/amazon-cloudwatch-metric-streams-additional-statistics/), including percentiles.
+
+Follow these steps to add percentiles to any metric available in the CloudWatch stream:
+
+1. On AWS, update the CloudWatch stream configuration (via API, CLI, or AWS Console) with the required percentiles in the `StatisticConfiguration` setting. For example, you can add p90, p95, and p99 percentiles to the `ELB latency metric (aws.elb.Latency)`.
```

Comment:
CloudWatch stream > CloudWatch Metric Stream?

Reply:
i think in this context it's referring to a stream, which iS something you create on the AWS side. you can have multiple metric streams that you create, and this all happens within the CloudWatch Metric Streams service. love product naming conventions lol